staticServer/staticServerConfig/config.go

```diff
@@ -0,0 +1,97 @@
+package staticServerConfig
+
+import (
+	"io/ioutil"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+// Path to yaml file, used for server configuration
+const (
+	CfgPath           = "../../staticServer/staticServerConfig/clientConfig.yaml"
+	DefaultPort       = "5080"
+	DefaultStaticPath = "../../staticServer/static"
+	DefaultPrefix     = "/static/"
+	DefaultServerAddr = "http://localhost:9000"
+)
+
+// Config structs yaml configuration
+type Config struct {
+	Port            string `yaml:"port"`
+	Host            string `yaml:"localhost"`
+	StaticFilesPath string `yaml:"staticFilesPath"`
+	Prefix          string `yaml:"pathPrefix"`
+	DynamicServer   string `yaml:"dynamicServerAddress"`
+}
+
+var serverConfig *Config
+
+func init() {
+	serverConfig = GetSettings(CfgPath)
+	serverConfig.setPath()
+	serverConfig.setPort()
+	serverConfig.setDynamicServer()
+}
+
+// GetSettings reads configuration file and stores values to struct variable
+func GetSettings(cfgPath string) *Config {
+	c := new(Config)
+	cfg, err := ioutil.ReadFile(cfgPath)
+	if err != nil {
+		log.Errorf("Failed to read config file %v", err)
+		return c
+	}
+	err = yaml.Unmarshal(cfg, c)
+	if err != nil {
+		log.Errorf("Failed to read config file %v", err)
```

Comment:
If you fail on unmarshaling, would it be correct to return `*Config` rather than error?